### DISCUSSION TITLE
WP_1.01-choisir-un-hebergeur-adapte

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#36] [Bonnes pratiques]
Lien vers la BP : https://github.com/cnumr/best-practices-wordpress/blob/main/fiches/WP_1.01-choisir-un-hebergeur-adapte.md

**Comment by dNicolle** [comment]
Cette fiche n'a personne d'attitré (Catherine n'ayant plus le temps pour ce projet).

**Comment by missfx** [comment]
Déjà il est important de faire référence à cette page dans la doc : https://fr.wordpress.org/about/requirements/

**Reply by dNicolle** [reply]
yes, merci

**Comment by marsaultgaetan** [comment]
On peut rajouter dans la liste des hébergeurs : 
- Datacampus : https://datacampus.fr/

**Reply by florinesueur** [reply]
Merci pour la proposition d'ajout, cependant pouvez-vous nous préciser les liens des pages où on peut "vérifier" les quelques critères énoncés comme les normes ISO.

**Reply by florinesueur** [reply]
@marsaultgaetan Avez-vous plus d'informations à ce sujet ?

**Comment by marsaultgaetan** [comment]
DATACAMPUS

-Normes ISO 14001 il y a quelques années : non reconduite au regard du temps à y accorder : priorisation RSE et innovation, voir suivant :

> Politiques et efficience énergétique :
- Refroidissement par immersion: https://datacampus.fr/immersion-cooling/
- PUE < 1,03
- Politique du bâti et achats responsables : https://datacampus.fr/datacenter-futuroscope/
- Entreprise à Mission : https://datacampus.fr/entreprise-a-mission/
- Politique sociale : https://positiveworkplace.fr/portfolio_category/positive-workplace/page/3/
- Compensation au profit direct de la biodiversité : https://www.createurdeforet.fr/entreprises/datacampus/
